src/neume.cpp

```diff
@@ -117,7 +117,13 @@ NeumeGroup Neume::GetNeumeGroup()
         }
         previous = current;
     }
-    return s_neumes[key];
+
+    try {
+        return s_neumes.at(key);
+    }
+    catch (std::out_of_range &e) {
+        return NEUME_ERROR;
+    }
```

Comment:
Why to you need to change this?

Reply:
s_neumes is now static const, so we must replace operator[] which only works for non-const maps. One way or the other we should check that the key is valid.

Reply:
OK. Then rather than a `try/catch` check with `.count` and return `NEUME_ERROR` if not found. We use `try/catch` only in quite specific places (e.g., input) and we should stick with this.

@JRegimbal does it work for you?

Reply:
This looks like it should be fine for us! @kemalkongar pinging you so other people are aware of this change.

Reply:
I guess the reason I preferred try/catch is that it is slightly more performant. Prechecking with find or count requires two binary searches per element access. It's not a big deal, since our map here is tiny small.